app/assets/stylesheets/custom.scss

```
@@ -79,6 +79,50 @@ aside {
   margin-top: 15px;
 }
 
+.stats {
+  overflow  : auto;
+  margin-top: 0;
+  padding   : 0;
+
+  a {
+    float      : left;
```

Bảo mấy lần r. Bỏ cái kiểu indent này đi
```suggestion
    float: left;
```

---

cái này vs code của e nó ăn extension gì á a ạ nên a bỏ qua cho e indent này nhé :))))